client/src/components/SpectatorCount.tsx

```diff
@@ -0,0 +1,18 @@
+import eyeIcon from "../assets/EyeIcon.svg";
+
+type SpectatorCountProps = {
+  count: number;
+};
+
+const SpectatorCount = ({ count }: SpectatorCountProps) => {
+  return (
+    <div className="fixed bottom-5 right-7 text-4xl text-white font-bold items-center flex">
+      <div className="float-left scale-90">
+        <img src={eyeIcon} alt="EyeIcon" />
```

Comment:
Alt text is meant to explain what is happening to people who have vision impairments, its meant to explain what is happening. Can we change this to "Spectators" to explain what the eye symbol means?